Conjur/conventions/git-tips-and-tricks.md

```diff
@@ -165,6 +167,29 @@ git push --force
 For more info on rebasing, especially on rebasing vs merging, see
 [here](https://git-scm.com/book/en/v2/Git-Branching-Rebasing).
 
+### From a fork
+
+If you are working from a fork, then from the command line in the root directory of the project on
+your local machine, set the upstream:
+```
+git remote add upstream https://github.com/org/repo
+```
+
+Fetch all the branches of the remote upstream project:
+```
+git fetch upstream
+```
+
+Rewrite your local branch to replay any commits from your local branch on top of the `upstream/master`:
+```
+git rebase upstream/master
+```
```

Comment:
I'd add here to do a `git log` just to make sure the branch looks as-expected before overwriting the master in our repo

Reply:
do you mean "before overwriting the master in your repo"?

Reply:
@izgeri correct. after the rebase but before the push.

Reply:
fixed - will need a squash merge